com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1540,21 +1497,5 @@ private IInternalMessageHandler CreateMessageHandler()
 
             return messageHandler;
         }
-
-        private void ProfilerBeginTick()
-        {
-            ProfilerNotifier.ProfilerBeginTick();
-        }
-
-        private void NotifyProfilerListeners()
-        {
-            ProfilerNotifier.NotifyProfilerListeners();
-            NetworkMetrics.DispatchFrame();
```

Comment:
Pretty sure we still need this part.

Reply:
Yeah, I just moved it to the original callsite for NotifyProfilerListeners

Reply:
Here's the line where it's still being called:
https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/1048/files#diff-d52d5b5b61ed193efdd8dc579c9919ac1493426df078c04d0aa0a4bd88cfaf96R940